crcsim/experiment/prepare.py

```diff
@@ -200,122 +204,54 @@ def transform(params):
     # scenario and the implementation scenario and vary diagnostic compliance.
-    initial_compliance = {
-        "fqhc1": (0.522, 0.593),
-        "fqhc2": (0.154, 0.421),
-        "fqhc3": (0.519, 0.568),
-        "fqhc4": (0.278, 0.374),
-        "fqhc5": (0.383, 0.572),
-        "fqhc6": (0.211, 0.392),
-        "fqhc7": (0.257, 0.354),
-        "fqhc8": (0.190, 0.390),
-    }
-    low_initial_stage_3_treatment_cost = 67_300
-    low_initial_stage_4_treatment_cost = 97_931
-    extra_low_initial_stage_3_treatment_cost = 50_000
-    extra_low_initial_stage_4_treatment_cost = 80_000
-
-    diagnostic_compliance_rates = {
-        "100Compliance": 1.0,
-        "80Compliance": 0.8,
-    }
 
-    FIT_cost_dict = {
-        "Public": 22,
-        "Patient+Public": 44,
+    compliance_rates = {
+        "NC": {
+            "initial": (0.097, 0.300),
+            "diagnostic": (0.444, 0.688),
+        },
+        # other items
     }
 
-    Colonoscopy_cost_dict = {
-        "Public": 912,
-        "Patient+Public": 1437,
+    costs = {
+        "public": {
+            "FIT": 22,
+            "Colonoscopy": 912,
+        },
+        #'Patient-Public": {"FIT": 44, "Colonoscopy": 1437,}
     }
 
     scenarios = []
 
-    for fqhc, screening_rates in initial_compliance.items():
-        for compliance, diagnostic_rate in diagnostic_compliance_rates.items():
-            for cost_category in (
-                FIT_cost_dict.keys() & Colonoscopy_cost_dict.keys()
-            ):  # Only common keys
-                FIT_cost = FIT_cost_dict[cost_category]
-                Col_cost = Colonoscopy_cost_dict[cost_category]
-                baseline = (
-                    Scenario(
-                        name=f"{fqhc}_{cost_category}_{compliance}_baseline",
-                        params=get_default_params(),
-                    )
-                    .transform(transform_initial_compliance(screening_rates[0]))
-                    .transform(transform_diagnostic_compliance(diagnostic_rate))
-                    .transform(transform_test_cost(Test.FIT, FIT_cost))
-                    .transform(transform_test_cost(Test.COLONOSCOPY, Col_cost))
-                )
-                scenarios.append(baseline)
-
-                implementation = (
-                    Scenario(
-                        name=f"{fqhc}_{cost_category}_{compliance}_implementation",
-                        params=get_default_params(),
-                    )
-                    .transform(transform_initial_compliance(screening_rates[1]))
-                    .transform(transform_diagnostic_compliance(diagnostic_rate))
-                    .transform(transform_test_cost(Test.FIT, FIT_cost))
-                    .transform(transform_test_cost(Test.COLONOSCOPY, Col_cost))
+    for site, rates in compliance_rates.items():
+        for cost_category, test_costs in costs.items():
+            FIT_cost = test_costs["FIT"]
+            Col_cost = test_costs["Colonoscopy"]
```

Comment:
Lowercase variable names is the convention in python.
```suggestion
            fit_cost = test_costs["FIT"]
            col_cost = test_costs["Colonoscopy"]
```